test/e2e/persistent_volumes.go

```diff
@@ -428,12 +470,82 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 		}
 	})
 
-	// Create an nfs PV that is *pre-bound* to a claim. Create a pod that
-	// contains the claim. Verify that the PV and PVC bind correctly and that
-	// the pod can write to the nfs volume.
-	It("should create a pre-bound PersistentVolume, Claim, and Pod that will test write access of the volume [Flaky]", func() {
+	// Create a claim first, then a nfs PV that matches the claim, and a
+	// pod that contains the claim. Verify that the PV and PVC bind
+	// correctly, and that the pod can write to the nfs volume.
+	It("should create a PVC and non-pre-bound PV: test write access [Flaky]", func() {
+
+		pv, pvc, err = createPVCPVnoPrebind(c, serverIP, ns)
+		if err != nil {
+			framework.Failf("%v", err)
+		}
+
+		pv, pvc, err = waitAndValidatePVandPVC(c, ns, pv, pvc)
```

Comment:
Everything from `waitAndValidatePVandPVC` to `deletePersistentVolume` looks the same in all three tests, Can you please make a separate function that would do these binding/pod/recycler checks instead of copying the code?